<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BorrowEditDialog</class>
 <widget class="QDialog" name="BorrowEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>351</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layout">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="margin" stdset="0">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="ok_btn">
      <property name="text">
       <string>编辑</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="spacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>131</width>
        <height>31</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="back_btn">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStackedWidget" name="stack_page">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <widget class="QWidget" name="add_page">
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>221</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="1">
       <widget class="QLineEdit" name="add_user_id_edit"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="add_book_id_edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="add_user_id_label">
        <property name="text">
         <string>用户ID</string>
        </property>
        <property name="buddy">
         <cstring>add_user_id_edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="add_borrow_day_num_label">
        <property name="text">
         <string>借书天数</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="add_borrow_day_num_spin_box">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="add_book_id_label">
        <property name="text">
         <string>书籍ID</string>
        </property>
        <property name="buddy">
         <cstring>add_book_id_edit</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="error_page">
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>312</width>
       <height>122</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="error_label">
        <property name="text">
         <string>异常原因</string>
        </property>
        <property name="buddy">
         <cstring>error_edit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="err_price_lable">
        <property name="text">
         <string>价格</string>
        </property>
        <property name="buddy">
         <cstring>err_price_edit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="err_price_edit"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QPlainTextEdit" name="error_edit"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>373</width>
       <height>83</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="err_book_id_lable">
          <property name="text">
           <string>书籍id</string>
          </property>
          <property name="buddy">
           <cstring>err_book_id_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="err_book_id_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="err_book_name_lable">
          <property name="text">
           <string>书籍名称</string>
          </property>
          <property name="buddy">
           <cstring>err_book_name_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="err_book_name_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="err_user_id_label">
          <property name="text">
           <string>用户ID</string>
          </property>
          <property name="buddy">
           <cstring>err_user_id_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="err_user_id_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="err_user_login_name_lable">
          <property name="text">
           <string>用户名</string>
          </property>
          <property name="buddy">
           <cstring>err_user_login_name_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="err_user_login_name_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="err_borrow_date_label">
          <property name="text">
           <string>借书日期</string>
          </property>
          <property name="buddy">
           <cstring>err_borrow_date_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="err_borrow_date_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="err_borrow_deadline_label">
          <property name="text">
           <string>还书日期</string>
          </property>
          <property name="buddy">
           <cstring>err_borrow_deadline_edit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="err_borrow_deadline_edit"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>ok_btn</sender>
   <signal>clicked()</signal>
   <receiver>BorrowEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>back_btn</sender>
   <signal>clicked()</signal>
   <receiver>BorrowEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>333</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
